buildSrc/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -39,6 +39,11 @@ class AllowDefaultMethodRule : AbstractRecordingSeenMembers() {
         // semver.
         continue
       }
+      if (change == JApiCompatibilityChange.METHOD_ABSTRACT_NOW_DEFAULT) {
```

Comment:
Interestingly, adding a default implementation to these methods caused japicmp to fail. As discussed in this [convo](https://github.com/siom79/japicmp/issues/201#issuecomment-372437436), changing a method from abstract to default is not a breaking change.